android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -753,6 +941,7 @@ static public BraveActivity getBraveActivity() {
     @Override
     public void onActivityResult (int requestCode, int resultCode,
                                   Intent data) {
+        Log.e("BraveVPN", "onActivityResult : BraveActivity");
```

Comment:
This is a draft debug message, I leave the comment just not to forget remove it when PR will be ready